doc/user_guide/dynamo0p3.rst

```diff
@@ -2998,6 +2998,26 @@ Sets a field *field2* equal (DoF per DoF) to another field
 
   field2(:) = field1(:)
 
+setval_random
+^^^^^^^^^^^^^
+
+**setval_random** (**field**)
+
+Fills all elements of a field *field* using a sequence of ``real``,
+pseudo-random numbers in the interval ``0 <= x < 1``::
+
+  do df = 1, ndofs
```

Comment:
I would suggest just using the array notation as in other `setval_` builtins as we specify in the introduction that the loop is over DoFs. E.g. `field(:) = RAND()`.

Reply:
I deliberately didn't do this to make it clear that each element of `field` is given a *different* random value.

Reply:
Fair enough, let's leave it like that.